pkg/collector/corechecks/system/cpu/cpu_windows_test.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build windows
+
+package cpu
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator/mocksender"
+	"github.com/stretchr/testify/mock"
+)
+
+var (
+	firstSample = []TimesStat{
+		{
+			CPU:    "cpu-total",
+			User:   1229386,
+			System: 263584,
+			Idle:   25496761,
+			Kernel: 25496761 + 263584,
+		},
+	}
+	secondSample = []TimesStat{
+		{
+			CPU:    "cpu-total",
+			User:   1229586,
+			System: 268584,
+			Idle:   25596761,
+			Kernel: 25496761 + 268584,
```

Comment:
```suggestion
			Kernel: 25596761 + 268584,
```

This will also invalidate the test below, the expected values will have to be re-calculated.

Reply:
Oops 🤦